vcard_convert.php

```diff
@@ -358,46 +379,50 @@ function parse_adr(&$node, &$vcard)
 		}
 		
 		// values not splitted by Contact_Vcard_Parse if key is like item1.ADR
-		if (strstr($home[0][0], ';'))
+		if (isset($home[0][0]) && strstr($home[0][0], ';'))
 		{
 			$temp = explode(';', $home[0][0]);
-			$vcard->home += array(
+			$vcard->home = array_merge($vcard->home, array(
```

Comment:
The add operator `+` is not equal to array_merge(): "for keys that exist in both arrays, the elements from the left-hand array will be used" whereas with array_merge, the second argument wins.

What's the problem with the add operator in PHP 8 (haven't tested it)?